agentMET4FOF/metrological_agents.py

```diff
@@ -198,3 +201,43 @@ def custom_plot_function(self, data, sender_agent, **kwargs):
         else:
             trace = go.Scatter()
         return trace
+
+class MetrologicalGeneratorAgent(MetrologicalAgent):
+    """An agent streaming a sine signal
```

Comment:
I am not sure about this docstring. Isn't `MetrologicalGeneratorAgent` a more general case of 

```suggestion
    """An agent streaming a signal
```

? We decided to set the default to a sine signal, but as of now I could use any instance of `MetrologicalDataStreamMET4FOF` as signal.